transform/mattermost-analytics/models/marts/web_app/community/community_tracks.sql

```diff
@@ -0,0 +1,38 @@
+{{
+    config({
+        "materialized": "incremental",
+        "cluster_by": ['event_date']
+    })
+}}
+
+WITH community_prod AS (
+    SELECT 
+        'stg_mm_telemetry_prod__tracks' AS _source_relation,
+        {{dbt_utils.star(ref('stg_mm_telemetry_prod__tracks'))}}
+        , received_at::date as received_at_date
+        , timestamp::date as event_date
+     FROM
+       {{ ref('stg_mm_telemetry_prod__tracks') }} 
+    WHERE server_id = '93mykbogbjfrbbdqphx3zhze5c'
+{% if is_incremental() %}
+    AND received_at_date >= (SELECT MAX(received_at_date) FROM {{ this }}) 
+    AND _source_relation = 'stg_mm_telemetry_prod__tracks'
```

Comment:
This expression must be inside the `SELECT MAX(...) ` sub-query. Please also check the comments in the previous PR: https://github.com/mattermost/mattermost-data-warehouse/pull/1192

Reply:
Addressed all the comments in the previous closed PR

Reply:
Moved this expression inside the sub query